src/components/hooks/use-website-status.js

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
+import { useCookieState } from './use-cookie-state'
+import { BinarySocketBase } from 'common/websocket/socket_base'
+import { getDateFromToday } from 'common/utility'
+
+const WEBSITE_STATUS_COUNTRY_KEY = 'website_status'
+
+export const useWebsiteStatus = () => {
+    const [website_status, setWebsiteStatus] = useCookieState(null, WEBSITE_STATUS_COUNTRY_KEY, {
+        expires: getDateFromToday(7),
+    })
+    const [is_loading, setLoading] = useState(true)
+
+    useEffect(() => {
+        setLoading(true)
+        if (!website_status) {
+            const binary_socket = BinarySocketBase.init()
+
+            binary_socket.onopen = () => {
+                binary_socket.send(JSON.stringify({ website_status: 1 }))
```

Comment:
Let's put '1' in a variable, that tells us, what happens when the status is '1'

Reply:
this is backend API convention, any api call will have the value 1, e.g payment_methods: 1, active_symbols: 1, etc.,